### PR TITLE
Add description about removing a page

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds description about removing a page

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/cms/page-duplicate.html

### CODE REVIEW

File: src/cms/page-duplicate.md [hunk 1]
```diff
@@ -52,3 +52,29 @@ _Save & Duplicate_
 |Save & New|Save and close the current page, and begin a new page.|
 |Save & Duplicate|Save and close the current page, and open a new duplicate copy.|
 |Save & Close|Save and close the current page, and return to the Pages grid.|
+
+## Delete a Page
+
+There are two ways to remove created page. You can remove it from _Pages_ grid, or from the _Edit_ page.
```
Comment (BarnyShergold) [inline src/cms/page-duplicate.md:58]:
```suggestion
There are two ways to remove a created page. You can remove it from _Pages_ grid or from the _Edit_ page.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Two small correction - nice addition

**Comment by EvheniiaKolkhiieva** [comment]
Hi @BarnyShergold , thank you for reviewing! Done ✅

**Review by BarnyShergold — APPROVED** [review]
That was a quick turnround!

File: src/_data/toc/content.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ pages:
     - label: Adding a New Page
       url: "/cms/page-add.html"
       children:
-      - label: Duplicating a Page
+      - label: Duplicating & Deleting a Page
```
Comment (leslietilling) [inline src/_data/toc/content.yml:32]: We don't use ampersands (although the UI unfortunately has some)

```suggestion
      - label: Duplicating and Deleting a Page
```

File: src/cms/page-duplicate.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Duplicating a Page
+title: Duplicating & Deleting a Page
```
Comment (leslietilling) [inline src/cms/page-duplicate.md:2]:
```suggestion
title: Duplicating and Deleting a Page
```

File: src/cms/page-duplicate.md [hunk 4]
```diff
@@ -52,3 +52,29 @@ _Save & Duplicate_
 |Save & New|Save and close the current page, and begin a new page.|
 |Save & Duplicate|Save and close the current page, and open a new duplicate copy.|
 |Save & Close|Save and close the current page, and return to the Pages grid.|
+
+## Delete a Page
```
Comment (leslietilling) [inline src/cms/page-duplicate.md:56]:
```suggestion
## Delete a page
```

File: src/cms/page-duplicate.md [hunk 5]
```diff
@@ -52,3 +52,29 @@ _Save & Duplicate_
...
+
+### Method 2: Remove a page from the edit page
+
+1. On the _Admin_ sidebar, go to **Content** > _Elements_ > **Pages**.
+
+1. Find the page to be deleted.
+
+1. In the _Actions_ column for the block, click **Select** and choose **Edit**.
+
+1. In the menu bar click <span class="btn">Delete Page</span>.
```
Comment (leslietilling) [inline src/cms/page-duplicate.md:78]:
```suggestion
1. In the button bar, click <span class="btn">Delete Page</span>.
```

**Review by leslietilling — COMMENTED** [review]
Just some style edits

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**